developer-docs-site/docs/index.md

```diff
@@ -81,7 +83,7 @@ Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentatio
 </div>
 </div>
 
-## SDKs and APIs
+## Write apps with our SDKs and APIs
```

Comment:
Remove "our". Also, "Develop apps" feels more friendly to developers, rather than "write" (thereby conveying the meaning the developing apps is more than writing code).

Reply:
Done!